node/src/chain_spec.rs

```diff
@@ -304,6 +305,114 @@ pub fn testnet_config() -> ChainSpec {
     )
 }
 
+/// Configuration for the mainnet
+/// TODO: Change the keys for initial authorities and master membership
```

Comment:
```suggestion
/// TODO:
///     Change the keys for initial authorities and master membership
///     Set sudo key to 0x0000...
///     Set built-in bootstrap node to a dns record that does not constain the string "testnet"
///     Reduce number of endowed accounts?
///     Update bootstrap node multi-addr to new format if needed.
```

Reply:
I like this suggestion except didn't understand the last point, which is the new format?

Reply:
Regarding the last point: https://github.com/docknetwork/dock-substrate/pull/59#discussion_r462349065

Regarding endowed accounts. Ok.

Reply:
Oh, what we have is already the new format.